lib/options.py

```diff
@@ -57,6 +57,14 @@ def __init__(self):
                 help = """List of test suites to look for tests in. Default: "" -
                 means find all available.""")
 
+        parser.add_argument(
+                "--verbose",
+                dest = 'is_verbose',
+                action = "store_true",
+                default = False,
+                help = """Print test output to log.
```

Comment:
It is about tests with TAP13 output. I guess it worth to mention in the help.

Reply:
Fixed
`"Print TAP13 test output to log"`